### ISSUE TITLE
Duplicator for GD

### ISSUE THREAD

**Issue by pol76** [AyeCode/geodirectory#2457]
An Add-on that duplicates CPT/Categories/Listings/Pages/Price Packages/Ads and so on, making building a Directory a lot faster and agile.

**Comment by koraykupe** [comment]
Indeed, I need especially for events. Some events are not recurring, but repeats in different places with mostly same data.
